From 398f4c450986028364be97b9f1b378d7eb7c38fc Mon Sep 17 00:00:00 2001
From: Aswin Murugan <quic_aswinm@quicinc.com>
Date: Mon, 24 Mar 2025 11:21:47 +0530
Subject: [PATCH] PENDING: Enable IO Expander for Display support in qcs9100

Add support to enable 8-pin IO Expander compatibility in IO
Expander driver.

Signed-off-by: Aswin Murugan <quic_aswinm@quicinc.com>
Upstream-Status: Pending
---
 drivers/gpio/pca953x_gpio.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpio/pca953x_gpio.c b/drivers/gpio/pca953x_gpio.c
index e84038f312..8402dc1bba 100644
--- a/drivers/gpio/pca953x_gpio.c
+++ b/drivers/gpio/pca953x_gpio.c
@@ -404,6 +404,7 @@ static const struct udevice_id pca953x_ids[] = {
 	{ .compatible = "ti,tca6408", .data = OF_953X(8, PCA_INT), },
 	{ .compatible = "ti,tca6416", .data = OF_953X(16, PCA_INT), },
 	{ .compatible = "ti,tca6424", .data = OF_953X(24, PCA_INT), },
+	{ .compatible = "ti,tca9538", .data = OF_953X(8, PCA_INT), },
 	{ .compatible = "ti,tca9539", .data = OF_953X(16, PCA_INT), },
 	{ .compatible = "ti,tca9554", .data = OF_953X(8, PCA_INT), },
 
-- 
2.34.1

